### DISCUSSION TITLE
Articles and posts about Miller

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#1159] [Show and tell]
Hi,
a thread to collect posts and articles about Miller 

This one on Smashing Magazine.

Just to point this out to you and always thank @johnkerl 

https://www.smashingmagazine.com/2022/12/guide-command-line-data-manipulation-cli-miller/

**Comment by leizi163163** [comment]


**Reply by aborruso** [reply]
> 你好，我现在不在，我上线后，定会给你回复的。

@johnkerl I do not have the privilege to delete this message. I think it's spam

**Comment by aborruso** [comment]
Another Miller post on hackaday cc @johnkerl (I have changed the subject)

Linux Fu: Miller the Killer Makes CSV No Pest
https://hackaday.com/2022/12/28/linux-fu-miller-the-killer-makes-csv-no-pest/
